rfps/rfp-consensus-visualization-tool.md

```diff
@@ -0,0 +1,123 @@
+# RFP: Consensus Visualization Tool
+
+**Name of Project:** `Consensus Visualization Tool`
+
+**Proposal Category:** `app-dev`
+
+## Project Description
+
+We are seeking proposals for an open source blockchain consensus visualization tool that demonstrates Filecoin's Expected Consensus process. There is currently a chain visualizer at http://user.kittyhawk.wtf:8000/ however it only shows the heaviest chain and does not include potential chain forks.
```

Comment:
`We are seeking proposals for an open source blockchain consensus visualization tool that demonstrates Filecoin's Expected Consensus process. There is currently a chain visualizer at http://user.kittyhawk.wtf:8000/ that visualizes chain activity on the go-filecoin devnets. However, it only shows the heaviest chain and does not include potential chain forks. We are specifically looking for a visualization tool that shows the main (heaviest) chain, and also other forks.`